doc/releases/migration-guide-3.7.rst

```diff
@@ -351,6 +351,11 @@ Controller Area Network (CAN)
     :c:func:`can_set_mode` during application startup instead of failing when :c:func:`can_recover`
     is called at a later point in time.
 
+Crypto
+======
+
+* The CSS driver has been deprecated on NXP lpc55s36
```

Comment:
shouldn't thing include details on how to migrate away?

Reply:
Unfortunately we do not have a migration... internal issue being discussed. It is just pulled.